### PR TITLE
Allows to configure logging for third-party libraries

### PR DESCRIPTION
**Pull request by mik-laj**
---
Make sure to mark the boxes below before creating PR: [x]

- [X] Description above provides context of the change
- [X] Unit tests coverage for changes (not needed for documentation changes)
- [X] Target Github ISSUE in description if exists
- [X] Commits follow "[How to write a good git commit message](http://chris.beams.io/posts/git-commit/)"
- [X] Relevant documentation is updated including usage instructions.
- [X] I will engage committers as explained in [Contribution Workflow Example](https://github.com/apache/airflow/blob/master/CONTRIBUTING.rst#contribution-workflow-example).

---
In case of fundamental code change, Airflow Improvement Proposal ([AIP](https://cwiki.apache.org/confluence/display/AIRFLOW/Airflow+Improvements+Proposals)) is needed.
In case of a new dependency, check compliance with the [ASF 3rd Party License Policy](https://www.apache.org/legal/resolved.html#category-x).
In case of backwards incompatible changes please leave a note in [UPDATING.md](https://github.com/apache/airflow/blob/master/UPDATING.md).
Read the [Pull Request Guidelines](https://github.com/apache/airflow/blob/master/CONTRIBUTING.rst#pull-request-guidelines) for more information.

### CODE REVIEW

File: airflow/config_templates/config.yml [hunk 1]
```diff
@@ -517,6 +517,14 @@
       type: string
       example: ~
       default: "task"
+    - name: extra_loggers
+      description: |
+        A comma\-separated list of third-party logger names that will be configure to print messages to
```
Comment (potiuk) [inline airflow/config_templates/config.yml]:
```suggestion
        A comma\-separated list of third-party logger names that will be configured to print messages to
```

**Review by potiuk — APPROVED** [review]

**Comment by ashb** [comment]
We have (had?) A config for setting the log level for FAB - should we roll that in to this new setting?

**Comment by mik-laj** [comment]
@ashb Logs from FAB have been with us for a very long time. Erasing them would be a breaking change. I also guess that adding them to the configuration was dictated by another use case. FAB_LOG_LEVEL is often used in production to have access to valuable information in the log. EXTRA_LOGGER_NAMES is intended mainly for developers. You can set this variable using environment variables to see more information.

File: airflow/config_templates/default_airflow.cfg [hunk 2]
```diff
@@ -283,6 +283,11 @@ dag_processor_manager_log_location = {AIRFLOW_HOME}/logs/dag_processor_manager/d
 # Default to use task handler.
 task_log_reader = task
 
+# A comma\-separated list of third-party logger names that will be configure to print messages to
```
Comment (mik-laj) [inline airflow/config_templates/default_airflow.cfg]:
```suggestion
# A comma\-separated list of third-party logger names that will be configured to print messages to
```

**Comment by potiuk** [comment]
Yeah. I like the simplicity of enabling 3rd-party loggers in this case. I think it's much better than having to roll out your full logging configuration to cover that.

**Pull request merged**